packages/paste-website/src/pages/foundations/localization/index.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: UX Localization Guidelines
+description: Localization stands for adapting a product, service or content to fit a specific market or country.
```

Comment:
suggestion: I would use "refers to" here as opposed to "stands for" here since localization is not an acronym or abstracted piece of language.
```suggestion
description: Localization refers to adapting a product, service or content to fit a specific market or country.
```

Reply:
Yeah, that is a much better word to use here. Thank you!

Reply:
Added!